ScaleformUI_Lua/src/LobbyMenu/Columns/PlayerListColumn.lua

```diff
@@ -75,6 +75,11 @@ function PlayerListColumn:AddPlayer(item)
 end
 
 function PlayerListColumn:RemovePlayer(item)
+    if item == nil then
+        print("^1[ERROR] PlayerListColumn:RemovePlayer() - item is nil");
```

Comment:
U might use an assertion in here to generate a Lua error.. this way devs are forced to fix their implementation